wpt-status/index.html

```diff
@@ -16,11 +16,12 @@
     margin-block-end: 0;
 }
 
+/*
 #header {
     display: flex;
     align-items: center;
     justify-content: space-between;
-}
+}*/
```

Comment:
please remove this if is not longer needed instead of commenting it out.

Reply:
I'm not 100% sure, this comes from the very first version of the site that went trough several redesigns before landing here. And it seems this is not needed anymore.